2D Simulation/game.html

```diff
@@ -20,180 +21,139 @@
+          <span class="popuptext" id="myPopup" style="font-size:12px; font-family:verdana">
+            Instructions:<br>
+            <ol>
+              <li>
+                Click “start” button.
+              </li>
+              <li>
+                Click the “show the construction” button to see what pattern needs to be made.
+              </li>
+              <li>
+                Instruct the AI to make pattern by typing in the text box and making gestures by double-clicking.
+              </li>
+              <li>
+                Press the "Incorrect Move" button if the AI does something you don't expect.
+              </li>
+              <li>
+                Try to get the highest score you can!
+              </li>
+            </ol>
+          </span>
+          <p id="helper"
+             class="helper"
+             style="font-size:12px; font-family:verdana">
+            (click again to close it)
+          </p>
         </div>
       </div>
     </div>
 
-    <div class = "gadgetWrapper">
-      <small>
-        <small>
-          <small>
-            <p class="taskQ"
-               id="taskQ"
-               style="font-size: 30px; position: absolute; top: 0px; left: 0%">
-              Blocksworld
-            </p>
-          </small>
-        </small>
-      </small>
-
-      <div class="instruction"
-           id="instruction"
-           style="font-family:verdana; position: absolute; top: 9%">
-        <label>
-          Enter instructions:
-          <input id="txt_instruction" type="text" />
-        </label>
-
-        <button class="buttonenter"
-                id="buttonEnter"
-                onclick="send_user_message_to_server(currentConfig); inputlength()">
-          Enter
-        </button>
-        <div class="rightInstruction" style="float: right;">
-          <div class="showChosen"
-               id="showChosen"
-               style="float: left; position: static; margin-right: 5px;">
-            <button class="buttonLike"
-                    style="vertical-align: middle;"
-                    onclick="showConstruction(finalBlocks)">
-              Show the Construction
-            </button>
-          </div>
-          <div id="buttonIncorrectDiv" style="float: right;">
-            <button id="buttonIncorrect"
-                    onclick="correctionUI.show_corrections_modal()"
-                    style="vertical-align: middle;">
-              Incorrect Move
-            </button>
-          </div>
+    <p id="taskQ">
+      Blocksworld
+    </p>
+    
+    <div class="instruction"
+         id="instruction">
+      <label>
+        Instructions:
+        <input id="txt_instruction" type="text" />
+      </label>
+      
+      <button class="button"
+              id="buttonEnter"
+              onclick="send_user_message_to_server(currentConfig); inputlength()">
+        Enter
+      </button>
+      <div class="rightInstruction" style="float: right;">
```

Comment:
I believe the `rightInstruction` class does not exist, so the class specifier does nothing. We could remove the class specifier or create the class and move the style elements there. There are a few "classes" where this happens.